main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -329,6 +337,32 @@ object Safety {
     case Predicate.Body.Loop(_, _, _) => Set.empty
   }
 
+  /**
+    * Variables that are bound in lattice position without `fix`.
```

Comment:
Free variables that occur in a lattice position in atom that is not marked with `fix`.